api/models/education.py

```diff
@@ -1,7 +1,9 @@
-from database import db
+from database.db import db
 
 class Education(db.Document):
+    classification = db.StringField(required=True)
```

Comment:
This has to be any phrase other than `type` because type is a Python native phrase

Reply:
I left frontend know during one of our PR reviews